### DISCUSSION TITLE
After recording audio using Mycroft/mimi-recording-studio

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#826] [General Q&A] [answered]
Hi folks and @thorstenMueller , I have recorded my audio using  Mycroft/mimi-recording-studio.
 What will be the next step ?

**Comment by thorstenMueller** [comment, marked as answer]
Hello @ManoBharathi93 .
Maybe my video tutorial on the process of creating a TTS voice is helpful: https://youtu.be/4YT8WZT_x48

**Reply by ghost** [reply]
Hello sir @thorstenMueller , Yes sir I have watched that video there you are using config.json file and MRS2LJSPEECH FILE in  prepared file directory. Can you share the prepared directory for me please ?

I am your subscribers sir Because of your youtube videos only I am passionate in machine learning
